src/templates/cmpapi.js

```diff
@@ -189,7 +258,8 @@
         var consentByVendorIdParam = parameter;
         image = document.createElement('img');
         image.src =
-          '<%-CONSENT_SERVER_PROTOCOL%>://<%-CONSENT_SERVER_HOST%>/<%-API_VERSION%>/set-consent?consentByVendorId=' +
+          window.location.protocol +
+          '//<%-CONSENT_SERVER_HOST%>/<%-API_VERSION%>/set-consent?consentByVendorId=' +
```

Comment:
I'd add a random parameter here (e.g. timestamp) to guarantee that request is not cached by CDN